ios/FluentUI/Table View/TableViewHeaderFooterView.swift

```diff
@@ -296,30 +289,53 @@ open class TableViewHeaderFooterView: UITableViewHeaderFooterView, TokenizedCont
     /// If `accessoryView` is set, the accessory button (if any) will be replaced by this custom view. Clients are responsible
     /// for the appearance and behavior of both the `accessoryView` and `leadingView`, including event handling and accessibility.
     @objc open func setup(style: Style, title: String, accessoryView: UIView, leadingView: UIView? = nil) {
-        setup(style: style, title: title, accessoryButtonTitle: "")
-        self.accessoryView = accessoryView
-        self.leadingView = leadingView
+        setupBase(style: style, title: title, accessoryView: accessoryView, leadingView: leadingView)
     }
 
     /// - Parameters:
     ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
     ///   - accessoryButtonTitle: Optional accessory button title string.
     @objc open func setup(style: Style, accessoryButtonTitle: String) {
-        setup(style: style, accessoryButtonTitle: accessoryButtonTitle, leadingView: nil)
+        setupBase(style: style, accessoryButtonTitle: accessoryButtonTitle)
     }
 
     /// - Parameters:
     ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
     ///   - title: The title string.
     @objc open func setup(style: Style, title: String) {
-        setup(style: style, title: title, accessoryButtonTitle: "")
+        setupBase(style: style, title: title)
     }
 
+    /// This is the base setup method. All other setup methods call this method with their parameters.
     /// - Parameters:
     ///   - style: The `TableViewHeaderFooterView.Style` used to set up the view.
+    ///   - title: The title string
+    ///   - attributedTitle: Title as an NSAttributedString for additional attributes.
     ///   - accessoryButtonTitle: Optional accessory button title string.
+    ///   - accessoryView: The optional custom accessory view in the trailing edge of this view.
     ///   - leadingView: An optional custom view that appears near the leading edge of the view.
-    private func setup(style: Style, accessoryButtonTitle: String, leadingView: UIView? = nil) {
+    private func setupBase(style: Style,
+                           title: String? = nil,
+                           attributedTitle: NSAttributedString? = nil,
+                           accessoryButtonTitle: String = "",
+                           accessoryView: UIView? = nil,
+                           leadingView: UIView? = nil) {
+        attributedTitleFont = nil
+        attributedTitleColor = nil
+
+        if let attributedTitle = attributedTitle {
+            titleView.attributedText = attributedTitle
+            titleView.isSelectable = true
+
+            let attributes = attributedTitle.attributes(at: 0, effectiveRange: nil)
+            attributedTitleFont = attributes[NSAttributedString.Key.font] as? UIFont ?? tokenSet[.textFont].uiFont
+            attributedTitleColor = attributes[NSAttributedString.Key.foregroundColor] as? UIColor ?? tokenSet[.textColor].uiColor
```

Comment:
This will only work if the attributed string has only a single color and font associated. I expect we would want to support any arbitrary `NSAttributedString`, right?

Reply:
Yeah, I think allowing that make sense. I took a similar approach to the `Label` component and its `isUsingCustomAttributedText` property. The one thing to note is that if an attributed string is provided, no additional token overrides can/will be applied. The string will solely rely on the attributes it was assigned when it was created.